src/components/views/rooms/LinkPreviewWidget.js

```diff
@@ -73,36 +73,13 @@ module.exports = React.createClass({
         this.unmounted = true;
     },
 
-    onImageClick: function(ev) {
-        var p = this.state.preview;
-        if (ev.button != 0 || ev.metaKey) return;
-        ev.preventDefault();
-        var ImageView = sdk.getComponent("elements.ImageView");
-
-        var src = p["og:image"];
-        if (src && src.startsWith("mxc://")) {
-            src = MatrixClientPeg.get().mxcUrlToHttp(src);
-        }
-
-        var params = {
-            src: src,
-            width: p["og:image:width"],
-            height: p["og:image:height"],
-            name: p["og:title"] || p["og:description"] || this.props.link,
-            fileSize: p["matrix:image:size"],
-            link: this.props.link,
-        };
-
-        Modal.createDialog(ImageView, params, "mx_Dialog_lightbox");
-    },
-
```

Comment:
yes, content type is there somewhere in the OG meta. i guess i don't care that much; i kinda liked being able to 'zoom' the preview image rather than jumping to the page if you clicked on the image, which is why I implemented it. let's see how much folks complain about it being removed.